### DISCUSSION TITLE
Chord mode alt keyboard view

### DISCUSSION THREAD

**Discussion by legsmechanical** [SynthstromAudible/DelugeFirmware#356] [Ideas]
Would propose modeling after the launchpad pro mk3 implementation. Ability to program custom chords and quickly play scale-appropriate triads and in-key notes. 

https://support.novationmusic.com/hc/en-gb/articles/360011206299-Launchpad-Pro-Mk3-Chord-Mode-Guide

**Comment by sapphire-arches** [comment]
https://drive.google.com/file/d/18S9YaR88RzGbJc3eaflH6dNYpyFURrpj/view the OXI one chord mode (page 58) may also be useful inspiration

**Comment by weavermedia** [comment]
@alter-alter has a chord memory demo on YouTube that might be relevant here too: https://www.youtube.com/watch?v=vCpqs-diMoA

**Comment by m-m-adams** [comment]
Included in 1.1, will probably be expanded later

**Comment by Dust-r** [comment]
I really like the new custom chords mode in 1.1; can all of the 16 side buttons have this functionality? Currently, only one row of 8 seems available at a time.
